### PR TITLE
Fix for Issue 61

### LINKED ISSUE #61: Add terms from ABCs of open source to glossary
Throughout #100DaysOfOSS, we've been doing a series on the [ABCs of Open Source](https://dev.to/bekahhw/series/23962). Not all of these terms are currently in our glossary. 

Add a term or all of the terms from the series if you like, just make sure that there aren't any repeats.

These terms will be added to the `10-glossary.md` file.

### PR DESCRIPTION
**Pull request by Abhishek-90**
## Description

This PR adds about quality assurance and bug to the glossary. 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [X] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Fixes #61 

## Mobile & Desktop Screenshots/Recordings
Screenshot 2023-10-09 133400
Screenshot 2023-10-09 133436

## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [X] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
NO

### CODE REVIEW

**Comment by Abhishek-90** [comment]
NOTE: 
1. The commit message is a bit misleading, I mentioned `pull request(PR)` instead of `bug`
2. This First commit is for reviewing purpose, once it is approved by the maintainers, I will be adding the remaining ABCs to the glossary as well.

File: 10-glossary.md
Comment (RitaDee) [inline 10-glossary.md:48]: I suggest to improve on that you state it like this:

"A bug refers to an error, flaw, or defect in code that adversely affects the proper functioning of the software. Open-source projects often depend on contributions from the community to identify and rectify these bugs."

**Review by RitaDee — COMMENTED** [review]
There is a typo error in the first line "A bug refers is an error".

File: 10-glossary.md
Comment (Abhishek-90) [reply 10-glossary.md]: Changes added.

File: 10-glossary.md
Comment (BekahHW) [inline 10-glossary.md:43]: A primary use of documentation is to explain how to implement and use the product.

Comment (Abhishek-90) [reply 10-glossary.md:43]: This is what I should replace the docs description with?

Comment (CBID2) [reply 10-glossary.md:43]: Yes @Abhishek-90

Comment (CBID2) [inline 10-glossary.md:43]:
```suggestion
This is an abbreviation for the word documentation. Its primarily use is to explain how to implement and use the product. Documentation also provides people information on how to contribute to OSS projects and expectations for contributors. Markdown, a text-based coding language is often used to create these forms of content. Want to see one in action? Check out the [Style Guide in our Pizza Verse repo](https://github.com/open-sauced/pizza-verse/blob/main/style-guide.md)!
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Abhishek-90. Here’s an idea how to implement Bekah’s suggestion.

**Review by RitaDee — COMMENTED** [review]
You can add more terms:

## OSS Projects

OSS stands for "Open-Source Software" projects. These are software projects where the source code is made available to the public, allowing anyone to view, use, modify, and contribute to the software.

## Markdown

Markdown is a lightweight markup language commonly used for creating formatted text documents. It is widely used for creating documentation and ReadMe files in software development due to its simplicity and readability.

File: 10-glossary.md
Comment (Abhishek-90) [reply 10-glossary.md:43]: updating docs glossary with this suggestion.

Comment (Abhishek-90) [reply 10-glossary.md]: Documention glossary updated with the suggestion given by @CBID2

**Comment by Abhishek-90** [comment]
> You can add more terms:

Suggested items added and committed.

File: 10-glossary.md
Comment (RitaDee) [inline 10-glossary.md:43]: In the sentence: "Its primarily use is to explain how to implement and use the product," there's a small typo. The word "primarily" should be "primary."

Correct sentence: "Its primary use is to explain how to implement and use the product."

**Review by RitaDee — CHANGES_REQUESTED** [review]
You’re almost there! You can make correction to the typo.

File: 10-glossary.md
Comment (Abhishek-90) [reply 10-glossary.md]: Suggested changes are done. I have also made one change on the next sentence. Let me know if that is valid according to you.

**Comment by RitaDee** [comment]
LGTM 🍕

File: 10-glossary.md
Comment (CBID2) [inline 10-glossary.md:55]:
```suggestion
OSS stands for "Open-Source Software" projects. These are software projects where the source code is made available to the public, allowing anyone to view, use and modify the software.
```

File: 10-glossary.md
Comment (CBID2) [inline 10-glossary.md:59]:
```suggestion
Markdown is a lightweight markup language commonly used for creating formatted text documents. It is widely used for creating documentation and README files in software development due to its simplicity and readability.
```

File: 10-glossary.md
Comment (CBID2) [inline 10-glossary.md:63]:
```suggestion
A style guide is a set of rules and conventions that define the preferred formatting, writing style, and visual elements used in documentation and other content. This helps maintain consistency and clarity across documents, making them easier to read and understand.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Made some suggestions for improvement @Abhishek-90

**Comment by Abhishek-90** [comment]
> Made some suggestions for improvement @Abhishek-90

Changes committed as per suggestions.

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Pull request merged**
